tests-beta/android/MASVS-STORAGE/MASTG-TEST-00x05.md

```diff
@@ -0,0 +1,28 @@
+---
+platform: android
+title: Sensitive Data Leaked via Notifications
+id: MASTG-TEST-00x05 // TODO replace with real ID
+apis: [NotificationManager]
+type: [static, dynamic]
+weakness: MASWE-0054
+prerequisites:
+- identify-sensitive-data
+profiles: [L1, L2]
+---
+
+## Overview
+
+This test case checks if the application leaks sensitive data via notifications to third parties.
+
+## Steps
+
+1. Run a static analysis tool such as @MASTG-TOOL-0110 on the app source to identify if the `POST_NOTIFICATIONS` permission is declared in the manifest file (for above Android API 33). That would be an indication that the app creates notifications.
```

Comment:
```suggestion
1. Reverse engineer the app (@MASTG-TECH-0017).
2. Run a static analysis tool such as @MASTG-TOOL-0110 on the reverse-engineered app to identify if the `POST_NOTIFICATIONS` permission is declared in the manifest file (for above Android API 33). This would indicate that the app creates notifications.
```

Reply:
We assume there's no access to the source code so reversing the app is necessary.